### ISSUE TITLE
NGINX and ModSec Phases

### ISSUE THREAD

**Issue by uxbod** [SpiderLabs/ModSecurity#953]
When using Atomicorp rules their IP threat intelligence do not trigger when in phase 2, yet if I update the rules to use phase 1 they work fine; why ? what phases does the NGINX implementation work with ?

**Comment by zimmerle** [comment]
Hi @uxbod,

Not sure how Atomicorp IP threat intelligence works. I am afraid I have to ask for the rule, otherwise it will be very difficult for me to tell.

I can tell the phases should work in a similar way. In order to check if variables are filled or not you may want to use the DebugLogs.

**Comment by uxbod** [comment]
Hello Filipe. 

This has never triggered: 

SecRule REMOTE_ADDR "@rbl threat2.atomicrbl.com." \ 
"phase:2,t:none,deny,status:403,log,auditlog,msg:'Atomicorp.com WAF Rules: Threat Intelligence Match for Spamming Source on Atomicorp Threat Intelligence RBL (TI-2). See this URL for details http://www.atomicrbl.com/lookup',severity:'3',setvar:ip.threat2=1,expirevar:ip.threat2=1800,id:355501,rev:2,tag:'no_ar'" 

and I started to wonder why when this one did: 

SecRule REMOTE_ADDR "@rbl xbl.spamhaus.org" \ 
"phase:1,t:none,deny,status:403,log,id:350000,rev:2,msg:'Global RBL Match: IP is on the xbl.spamhaus.org Blacklist (Report False Positives to www.spamhaus.org)',severity:'3'" 

so I changed the phase to be :1 and now it is hitting. 

Thanks, Phil
